### PR TITLE
[DOC] Add more detail about SSO using OIDC

### PR DESCRIPTION
**Pull request by fdlk**
#### Checklist
- [N.A.] Functionality works & meets specifications
- [N.A.] Code reviewed
- [N.A.] Code unit/integration/system tested
- [N.A.] Migration step added in case of breaking change
- [x] User documentation updated
- [N.A.] (If you have changed REST API interface) view-swagger.ftl updated
- [N.A.] Test plan template updated
- [x] Clean commits
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: docs/guide-authentication.md [hunk 1]
```diff
@@ -58,17 +58,90 @@ Once enabled the sign in dialog will display additional sign in options:
...
-### Configuring single sign-on   
+## Configuring single sign-on   
 As administrator single sign-on configuration consists of the following one-time steps:
-1. Configuring one or more OpenID Connect client
-2. Modify authentication settings
-
-#### Configuring OpenID Connect clients
+1. Register MOLGENIS with the ID providers
+2. Configure one or more OpenID Connect clients, one for each identity provider
+2. Modify the authentication settings, activating clients for the identity providers.
```
Comment (dennishendriksen) [inline docs/guide-authentication.md]: 3.

File: docs/guide-authentication.md [hunk 2]
```diff
@@ -58,17 +58,90 @@ Once enabled the sign in dialog will display additional sign in options:
...
+| Token URI          | `https://<server>/auth/realms/<realm>/protocol/openid-connect/token` |
+| JWKS URI           | `https://<server>/auth/realms/<realm/protocol/openid-connect/certs` |
+| User info URI      | `https://<server>/auth/realms/<realm>/protocol/openid-connect/userinfo` |
+| Username attribute | `sub` |
+
+(See `https://<server>/auth/realms/<realm>/.well-known/openid-configuration`)
+
+### Activating OpenID Connect clients
 Configured OpenID Connect clients can be selected in the 'Authentication settings' part of the 'Settings manager' plugin. Activating clients
 requires 'Allow users to sign up' to be set to 'Yes' and 'Sign up moderation' to be set to 'No'.
```
Comment (Fernananas) [inline docs/guide-authentication.md]: And in 'Authentication server' one or more OpenID Connect clients needs to be added. Only the clients that are created in entity 'OIDC client' can be selected.
autentication

File: docs/guide-authentication.md [hunk 3]
```diff
@@ -58,17 +58,90 @@ Once enabled the sign in dialog will display additional sign in options:
...
+`https://<serverURL>/login/oauth2/code/<registrationID>`
+The registration ID is an ID that you assign to the provider when you register the
+OIDC client.
+*  Since the user will be authenticating with the ID provider in the browser,
+the provider may allow you to add some branding to the login page,
+such as a logo or styling.
+
+### Registering an ID provider with MOLGENIS
+MOLGENIS uses an OpenID Connect client to communicate with the ID provider.
 OpenID Connect clients can be configured by adding entities to the 'OIDC client' entity type (e.g. using the data explorer, importer or REST API).
```
Comment (Fernananas) [inline docs/guide-authentication.md]: SOPoidc
Example OIDC client, ID and secret are crossed out.

File: docs/guide-authentication.md [hunk 4]
```diff
@@ -58,17 +58,90 @@ Once enabled the sign in dialog will display additional sign in options:
 Selecting e.g. Google will redirect the browser to their website so the user can authenticate there and return to MOLGENIS once authentication has successfully completed.
 The permissions you have once authenticated are the default user permissions set by an administrator.   
 
-### Configuring single sign-on   
+## Configuring single sign-on
```
Comment (Fernananas) [inline docs/guide-authentication.md]: NOTE: only works if on server ntp is installed and running.

Comment (fdlk) [reply docs/guide-authentication.md]: I've added a little explanation of that at the end

File: docs/guide-authentication.md (hunk 3 above)
Comment (fdlk) [reply docs/guide-authentication.md]: I thought about this a bit more, and nowadays the best way to find an up to date version of the endpoints is the configuration endpoint. Updated the table below with a couple of example endpoints.

Comment (fdlk) [reply docs/guide-authentication.md]: Added the MITRE / perun config endpoint to the table

**Pull request merged**